lib/icinga/command.hpp

```diff
@@ -40,6 +40,11 @@ class Command : public ObjectImpl<Command>
 	//virtual Dictionary::Ptr Execute(const Object::Ptr& context) = 0;
 
 	void Validate(int types, const ValidationUtils& utils) override;
+
+protected:
+	void CommandChangeHandler(void);
+
+	virtual void OnConfigLoaded(void) override;
```

Comment:
`virtual` is redundant here because `override` can only be specified for `virtual` methods anyway.